forge-gui/res/cardsfolder/upcoming/sanwell_avenger_ace.txt

```diff
@@ -0,0 +1,14 @@
+Name:Sanwell, Avenger Ace
+ManaCost:1 W
+Types:Legendary Creature Human Pilot
+PT:3/1
+R:Event$ DamageDone | Prevent$ True | ValidTarget$ Creature.Self |  IsPresent$ Artifact.Creature+YouCtrl+attacking | Description$ As long as an artifact creature you control is attacking, prevent all damage that would be dealt to CARDNAME.
+T:Mode$ Taps | ValidCard$ Card.Self | Execute$ DBExileTopCard | TriggerDescription$ Whenever CARDNAME becomes tapped, exile the top six cards of your library. You may cast a Vehicle or artifact creature spell from among them. Then put the rest on the bottom of your library in a random order.
+SVar:DBExileTopCard:DB$ Dig | Defined$ You | DigNum$ 6 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STPlay | Triggers$ ReturnonPlay | RememberObjects$ Remembered | ForgetOnMoved$ Exile | Duration$ Permanent
+SVar:STPlay:Mode$ Continuous | EffectZone$ Command | AffectedZone$ Exile | Affected$ Vehicle.IsRemembered,Artifact.Creature+IsRemembered | MayPlay$ True | Description$ You may play one those cards.
```

Comment:
e | Description$ You may play one OF those cards.

Reply:
Description is no longer needed on the latest script. Thanks for the comments though